### ISSUE TITLE
about the  weight loss of 

### ISSUE THREAD

**Issue by kekedan** [NVIDIA/vid2vid#78]
Hi,I'm   confused about  weight loss，the weight loss is defined as 
  ################## weight loss ##################
            loss_W = torch.zeros_like(weight)
            if self.opt.no_first_img:
                dummy0 = torch.zeros_like(weight)
                loss_W = self.criterionFlow(weight, dummy0, conf_ref) 
1. why there use dummy0 ?
2. weight is used in model_g as  follow：
img_final = img_raw * weight_ + img_warp * (1-weight_)
so the weight loss is aim to reduce the weight of  img_raw,why not  img_warp ?
can you explain these? than you !
